ProcessLib/InitialCondition.h

```diff
@@ -45,8 +45,8 @@ class UniformInitialCondition : public InitialCondition
 	UniformInitialCondition(double const value) : _value(value)
 	{
 	}
-
-	virtual double getValue(MeshLib::Node const&) const override
+	virtual double getValue(MeshLib::Node const&,
+	                        int const /* tuple_size */) const override
```

Comment:
I guess, `tuple_size` is the wrong name. Maybe that is what you meant in your comment above.
Anyway, I think, also for uniform ICs, different components can have different values. Abd that should be reflected in this class.